.github/workflows/set-version.yaml

```diff
@@ -26,15 +26,15 @@ jobs:
         sed -i -e 's/const versionMajorMinor = ".*"/const versionMajorMinor = "${{ github.event.client_payload.core_major_minor }}"/g' tests/baselines/reference/api/typescript.d.ts
         sed -i -e 's/const versionMajorMinor = ".*"/const versionMajorMinor = "${{ github.event.client_payload.core_major_minor }}"/g' tests/baselines/reference/api/tsserverlibrary.d.ts
         sed -i -e 's/const version\(: string\)\{0,1\} = .*;/const version = "${{ github.event.client_payload.package_version }}" as string;/g' src/compiler/corePublic.ts
-        npm ci
+        npm install

```

Comment:
This and the `package-lock` change are not strictly required, however, before we had been leaving release branches in an unclean state by modifying the version in package.json but not the lockfile.

Reply:
I guess I don't understand why this would be necessary.

Reply:
Okay, we talked about it. You mentioned modern npm consults the lockfile.

Just make sure that the version of npm on this action is modern enough.

Reply:
I'm just going to revert it, this was overkill. I just want LKG to work and that's it. the rest can stay for now unless it breaks